### DISCUSSION TITLE
Random Discord Ban

### DISCUSSION THREAD

**Discussion by Poisonlocket** [Rapptz/discord.py#9701] [Q&A] [answered]
hi there, i've recently joined the discordpy discord and asked a few questions regarding the library. Now all of a sudden im Banned. Is there any way i can get to know the reason or appeal this?  It just really seems confusing, maybe there has been an Error

**Comment by Rapptz** [comment, marked as answer]
You got autobanned for spamming. I've unbanned you but be mindful of sending multiple messages over a short period.

**Reply by Poisonlocket** [reply]
Oh wow, thanks i didnt know spamming rules were that close. No worries wont happen again

**Comment by Shitanshukumar607** [comment]
Hey there, I was suddenly banned from the discord.py server maybe because I pinged 5 ppl can you unban me ? My id - stargaming7788

**Reply by ika2kki** [reply]
i've unbanned you but try to be mindful with mentions (the limit is 4) in the future.
ps. please don't ask for unbans here (or anything regarding moderation) and instead try to DM a mod~ a mutual server shared with most d.py mods is [Discord API](https://discord.gg/discord-api)

**Reply by Shitanshukumar607** [reply]
Ty
